modules/ROOT/pages/custom-ingress-configuration.adoc

```diff
@@ -0,0 +1,52 @@
+= Defining a Custom Ingress Configuration
+
+[NOTE]
+====
+The tasks outline in this topic must be performed by an IT administrator
+====
+
+Runtime Fabric on Self-managed Kubernetes enables you to edit a template file that specifies a custom Ingress configuration. Using this file you can define annotations, rewrite rules, the ingressClassName, and more.
+
+The template file is a Kubernetes ConfigMap file. After customizing the template, you can use the `rtfctl apply` command-line utility to apply the custom Ingress configuration to your Kubernetes cluster.
```

Comment:
we might want to give the full sub-command `rtfctl apply ingress-configmap`

Reply:
Fixed.